views/_system_large.erb

```diff
@@ -6,7 +6,7 @@
         '<img src="<%= ganglia.report_moab_nodes.size(800,250).png %>" class="img-responsive center-block updateable" id="large-image<%= @count %>" >');" >
             <div class="panel-heading"><h4 class="panel-title">Nodes</h4></div>
             <div class="panel-body">
-              <img class="img-responsive center-block updateable" alt="Graph" src="<%= ganglia.small.report_moab_nodes.png %>" onclick=" $('<%= #large-image#{@count} %>').replaceWith(
+              <img class="img-responsive center-block updateable" alt="Graph" src="<%= ganglia.small.report_moab_nodes.png %>" onclick=" $('#large-image<%= @count %>').replaceWith(
```

Comment:
FYI the reason that this doesn't work:

```erb
onclick=" $('<%= #large-image#{@count} %>')
```

is that the ruby expression is missing quotes:

```erb
onclick=" $('<%= "#large-image#{@count}" %>')
```